packages/core/src/auth/resetPassword.tsx

```diff
@@ -1,7 +1,9 @@
 import { IResetPassword } from './types'
 import * as Yup from 'yup'
+import { yupResolver } from '@hookform/resolvers/yup'
 import { axios, useMutation } from '../api'
-import { useFormik } from 'formik'
+import { useForm } from 'react-hook-form'
```

Comment:
I know you are going to hate me lol, but could we organize the imports?
// lib imports:
import * as Yup from 'yup'
...
< ---- space ---- >
// relative imports:
import { axios, useMutation } from '../api'
...

Reply:
Seems like this pretties plugin would do what we want https://www.npmjs.com/package/@trivago/prettier-plugin-sort-imports

Reply:
I lik

> unrelated to this PR, noticed we repeat this prop type definition with different core hooks:
> 
> ```
>   validationSchema = signUpValidationSchema,
>   defaultValues = defaultInitialValues,
>   onError,
>   onSuccess,
> }: {
>   validationSchema?: any
>   defaultValues?: any
>   // eslint-disable-next-line @typescript-eslint/ban-types
>   onError?: Function
>   // eslint-disable-next-line @typescript-eslint/ban-types
>   onSuccess?: Function
> })
> ```
> 
> we prolly should extract and reuse

good catch, that's should be fixed on https://silverlogic.atlassian.net/browse/BA-902 @Janekk

Reply:
that's coool